projects/hipfft/shared/accuracy_test.h

```diff
@@ -352,41 +352,43 @@ inline void execute_gpu_fft(Tparams&              params,
     // Execute the transform:
     auto fft_status = params.execute(pibuffer.data(), pobuffer.data());
     if(fft_status != fft_status_success)
-        throw std::runtime_error("plan execution failure");
-    // work around potential problem of following hipMemcpy
-    // not properly waiting for rocFFT's kernels to finish
-    if(hipDeviceSynchronize() != hipSuccess)
-        throw std::runtime_error("hipDeviceSynchronize after execute failed");
+        throw std::runtime_error("rocFFT plan execution failure");
```

Comment:
```suggestion
        throw std::runtime_error("hipFFT plan execution failure");
```
Does it make sense to refer to "rocFFT plan" at all in this case? There may be none involved depending on the backend.